Bug 1774377: [DOCS] No mention for how to upgrade RHEL compute node from v4.1 to v4.2 (#18328)
openshift/openshift-docs: modules/rhel-compute-updating.adoc

You must not enable firewalld later. If you do, you cannot access {product-title} logs on the worker.
====

> . Enable the repositories required by {product-title} 4.2, if you upgrade from 4.1 to 4.2.

Reviewer: ```suggestion
. Enable the repositories that are required for {product-title} {product-version}:
```

> .. Preparing the machine to run the playbook

Reviewer: ```suggestion
.. On the machine that you run the Ansible playbooks, update the required repositories:
```

+
----
# subscription-manager repos --disable=rhel-7-server-ansible-2.7-rpms  \
                             --disable=rhel-7-server-ose-4.1-rpms \
                             --enable=rhel-7-server-ansible-2.8-rpms \
                             --enable=rhel-7-server-ose-4.2-rpms
----

> .. Update the required packages, including Openshift-Ansible:

Reviewer: ```suggestion
.. On the machine that you run the Ansible playbooks, update the required packages, including `openshift-ansible`:
```

+
----
# yum update openshift-ansible openshift-clients
----

> .. Preparing a RHEL compute node

Reviewer: ```suggestion
.. On each RHEL compute node, update the required repositories:
```
